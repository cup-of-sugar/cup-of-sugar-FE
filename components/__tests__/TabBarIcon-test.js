import React from 'react';
import TabBarIcon from '../TabBarIcon';
import { Ionicons } from '@expo/vector-icons';

import renderer from 'react-test-renderer';

test('should render component', () => {
  const props = {
    name: 'ios-add',
    focused: true,
  };
  const tree = renderer.create(<TabBarIcon {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('should render component', () => {
  const props = {
    name: 'ios-add',
    focused: false,
  };
  const tree = renderer.create(<TabBarIcon {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
