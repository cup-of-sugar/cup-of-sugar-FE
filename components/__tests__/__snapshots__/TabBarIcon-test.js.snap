// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `<Text />`;

exports[`should render component 2`] = `<Text />`;
